docs/guides/runbooks_steps_firehydrant.md

```diff
@@ -0,0 +1,779 @@
+        }
+      }
+    })
+
+    automatic = true
+    repeats   = false
+  }
+}
+```
+
+### Assign a Role - Steps Argument Reference
+
+* `action_id` - (Required) The ID of the runbook action for the step.
+* `config` - (Required) JSON string representing the configuration settings for the step.
+  Use [Terraform's jsonencode](https://www.terraform.io/language/functions/jsonencode)
+  function so that [Terraform can guarantee valid JSON syntax](https://www.terraform.io/language/expressions/strings#generating-json-or-yaml).
+* `name` - (Required) The name of the step.
+* `automatic` - (Optional) Whether this step should automatically execute.
+* `repeats` - (Optional) Whether this step should repeat. Defaults to `false`.
+  When this value is `true`, `repeats_duration` _must_ be provided.
+* `repeats_duration` - (Optional) How often this step should repeat in ISO8601.
+  Example: PT10M [Format Spec](https://www.digi.com/resources/documentation/digidocs/90001437-13/reference/r_iso_8601_duration_format.htm)
+  This value _must_ be provided if `repeats` is `true`. This value _must not_ be provided if `repeats` is `false`.
+
+The `config` block supports:
+
+* `role` - (Required) The role to assign.
+* `user` - (Optional) The user to assign the role to.
+
+The `role` block supports:
+
+* `label` - (Required) The name of the role.
+* `value` - (Required) The ID of the role.
+
+The `user` block supports:
+
+* `label` - (Required) The name of the user.
+* `value` - (Required) The type and ID of the user.
+
+The `value` block supports:
+
+* `type` - (Required) The type of user. Valid values are `incident_opener`, 
+  `external_schedule`, or `user`.
+* `external_schedule_id` - (Optional) The ID of the external schedule type user. 
+  This value _must not_ be provided if `type` is `incident_opener` or `user`.
+* `user_id` - (Optional) The ID of the user type user.
+  This value _must not_ be provided if `type` is `incident_opener` or `external_schedule`.
+
+## Assign a Team
+
+The [FireHydrant **Assign a Team** runbook step](https://support.firehydrant.com/hc/en-us/articles/360061144571-Communicating-with-teams-using-Runbook-steps)
+allows teams to be assigned to an incident automatically.
+
+### Assign a Team - Example Usage
+
+Basic usage:
+```hcl
+data "firehydrant_runbook_action" "firehydrant_assign_a_team" {
+  integration_slug = "patchy"
+  slug             = "assign_a_team"
+  type             = "incident"
+}
+
+resource "firehydrant_team" "example_team" {
+  name        = "example-team"
+  description = "This is an example team"
+}
+
+resource "firehydrant_runbook" "firehydrant_assign_a_team_runbook" {
+  name = "assign-a-team-runbook"
+
+  steps {
+    name      = "Assign A Team"
+    action_id = data.firehydrant_runbook_action.firehydrant_assign_a_team.id
+
+    config = jsonencode({
+      team = {
+        label = data.firehydrant_team.example_team.name
+        value = data.firehydrant_team.example_team.id
```

Comment:
These should directly reference the team resource, not a data block

Reply:
whoops! good catch, ty